src/ex4/client/src/components/ButtonComp.jsx

```diff
@@ -0,0 +1,20 @@
+import "../App.css";
+function ButtonComp({ imgSrc, className, imgClassName, alt, onClick }) {
+	const bulidButton = () => {
```

Comment:
react funcional components are functions. no need to add another function inside and just call it, just return directly the div